Sources/FodyRunner.UnityIntegration/EditorWeaver.cs

```diff
@@ -1,6 +1,7 @@
 ﻿namespace Malimbe.FodyRunner.UnityIntegration
 {
     using System;
+    using System.Reflection;
```

Comment:
This needs aliasing as it is causing a ambiguous conflict.

so something like:

`using Reflection = System.Reflection;`